tests/integration/cmn/client/acl.go

```diff
@@ -0,0 +1,119 @@
+	"github.com/anoideaopen/foundation/test/integration/cmn/client"
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/golang/protobuf/proto"
+	"github.com/hyperledger/fabric/integration/nwo"
+	"github.com/hyperledger/fabric/integration/nwo/commands"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+	"github.com/onsi/gomega/gexec"
+)
+
+const MultisignKeyDelimeter = "/"
+
+func CheckUserMultisigned(network *nwo.Network, peer *nwo.Peer, user *UserFoundationMultisigned) {
+	Eventually(func() string {
+		sess, err := network.PeerUserSession(peer, "User1", commands.ChaincodeQuery{
+			ChannelID: cmn.ChannelAcl,
+			Name:      cmn.ChannelAcl,
+			Ctor:      cmn.CtorFromSlice([]string{"checkKeys", user.PublicKey()}),
+		})
+		Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit())
+		Expect(sess.ExitCode()).To(Equal(0))
+		if sess.ExitCode() != 0 {
+			return fmt.Sprintf("exit code is %d: %s, %v", sess.ExitCode(), string(sess.Err.Contents()), err)
+		}
+
+		out := sess.Out.Contents()[:len(sess.Out.Contents())-1] // skip line feed
+		resp := &pb.AclResponse{}
+		err = proto.Unmarshal(out, resp)
+		Expect(err).NotTo(HaveOccurred())
+		if err != nil {
+			return fmt.Sprintf("failed to unmarshal response: %v", err)
+		}
+
+		addr := base58.CheckEncode(resp.GetAddress().GetAddress().GetAddress()[1:], resp.GetAddress().GetAddress().GetAddress()[0])
+		if addr != user.AddressBase58Check {
+			return fmt.Sprintf("Error: expected %s, received %s", user.AddressBase58Check, addr)
+		}
+
+		return ""
+	}, network.EventuallyTimeout, time.Second).Should(BeEmpty())
+}
+
+// AddMultisig adds multisigned user
+func AddMultisig(network *nwo.Network, peer *nwo.Peer, orderer *nwo.Orderer, n int, user *UserFoundationMultisigned) {
+	ctorArgs := []string{"addMultisig", strconv.Itoa(n), client.NewNonceByTime().Get()}
+	publicKeys, sMsgsByte, err := user.Sign(ctorArgs...)
+	var sMsgsStr []string
+	for _, sMsgByte := range sMsgsByte {
+		sMsgsStr = append(sMsgsStr, hex.EncodeToString(sMsgByte))
+	}
+	ctorArgs = append(append(ctorArgs, publicKeys...), sMsgsStr...)
+	sess, err := network.PeerUserSession(peer, "User1", commands.ChaincodeInvoke{
+		ChannelID: cmn.ChannelAcl,
+		Orderer:   network.OrdererAddress(orderer, nwo.ListenPort),
+		Name:      cmn.ChannelAcl,
+		Ctor:      cmn.CtorFromSlice(ctorArgs),
+		PeerAddresses: []string{
+			network.PeerAddress(network.Peer("Org1", "peer0"), nwo.ListenPort),
+			network.PeerAddress(network.Peer("Org2", "peer0"), nwo.ListenPort),
+		},
+		WaitForEvent: true,
+	})
+	Expect(err).NotTo(HaveOccurred())
+	Eventually(sess, network.EventuallyTimeout).Should(gexec.Exit(0))
+	Expect(sess.Err).To(gbytes.Say("Chaincode invoke successful. result: status:200"))
+
+	CheckUserMultisigned(network, peer, user)
+}
+
+// ChangeMultisigPublicKey changes public key for multisigned user by validators
+func ChangeMultisigPublicKey(network *nwo.Network, peer *nwo.Peer, orderer *nwo.Orderer,
```

Comment:
very long list of arguments. I'd reformat it placing one arg per line. like this:
func foo(
   arg1 string,
   arg2 int,
   ...
) error {}